sentimentanalysis/test_tweetSentiment.py

```diff
@@ -0,0 +1,167 @@
+from unittest import TestCase
+from sentimentanalysis.analyzer import TweetSentiment
+
+
+class TestTweetSentiment(TestCase):
+
+    def test_polarity_scores(self):
+        """
+        Tests a sentiment containing the word but in the middle of the sentence (both upper and lower cases)
+        1. lower case 'but'
+        2. upper case 'BUT'
```

Comment:
Should we also check for cases like 'buT' or 'bUt' etc?
-> would be covered in test_capitalized anyways